.storybook/stories/11-KaraokeMode.stories.tsx

```diff
@@ -54,6 +53,12 @@ export const KaraokeMode: React.FC<KaraokeModeProps> = args => {
 		}
 	}, [getCurrentTimePart, setCurrentPart]);
 
+	const findUpdates = (e: TimeUpdateEvent) => {
```

Comment:
See warning: `The 'findUpdates' function makes the dependencies of useEffect Hook (at line 81) change on every render. Move it inside the useEffect callback. Alternatively, wrap the definition of 'findUpdates' in its own useCallback() Hook`